packages/neutron-sdk/src/interchain_txs/helpers.rs

```diff
@@ -1,21 +1,30 @@
-use cosmos_sdk_proto::cosmos::base::abci::v1beta1::{MsgData, TxMsgData};
+use cosmos_sdk_proto::{
+    cosmos::base::abci::v1beta1::{MsgData, TxMsgData},
+    traits::Message,
+};
 use cosmwasm_std::{Binary, StdError, StdResult};
-use prost::{DecodeError, Message};
 
 /// Decodes acknowledgement into `Vec<MsgData>` structure
 pub fn decode_acknowledgement_response(data: Binary) -> StdResult<Vec<MsgData>> {
-    let msg_data: Result<TxMsgData, DecodeError> = TxMsgData::decode(data.as_slice());
+    let msg_data: Result<TxMsgData, _> = TxMsgData::decode(data.as_slice());
     match msg_data {
         Err(e) => Err(StdError::generic_err(format!(
             "Can't decode response: {}",
             e
         ))),
-        Ok(msg) => Ok(msg.data),
+        Ok(msg) => Ok(msg
+            .msg_responses
```

Comment:
this field should only be used starting from sdk46, currently neutron uses sdk45, hence we have to use deprecated field msg.data for a while, until sdk47 upgrade is live